results_calculator.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+from FileReader import FileReader
+
+# Part (a) calculations
+def recovered_over_total(args):
```

Comment:
Let's implement this whole module as a class.

Reply:
what do you mean?...I have used FileReader as a module of FileReader class here...

Reply:
Just like that, make a module out of this results_calculator file as well. For now it's just a file with functions in it. Turn it into a file with a class in it that has these functions.

Reply:
well now the functions recovered_over_total(args) and measures_death_rate(args) are not recognized after importing the class made in results_calculator